packages/components/addon/components/hds/code-block/title.hbs

```diff
@@ -0,0 +1,8 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Text::Body @size="200" @tag="p" @weight="semibold" class="hds-code-block__title" ...attributes>
```

Comment:
Where is the title taking the color from? if it's inherited, I would suggest to declare it explicitly in the CSS.

Also, since the `Title` is yielded (to allow structured content, according to the PR description, we should handle also the styling for it, like we do for the `Description` element.

In that case, please add a use case to the showcase with structured content, possibly beyond the simple `<a>` case, for example I would add `<strong>`, `<em>`, `<code>`

Reply:
This was indeed inherited from the component root. Updated in 7d39f55735aae014b9f8a28f71d4300d3a033552 to explicitly set color for title and description and apply basic elements styles on both of them.